### DISCUSSION TITLE
Installing MOOSE - libquadmath

### DISCUSSION THREAD

**Discussion by sinaheidari22** [idaholab/moose#16238] [Q&A General] [answered]
Hello everyone, 

I had MOOSE installed on ubuntu 16.04 version. Working perfect, I updated the MOOSE, it does not work anymore. I changed ubuntu to 20.04 version, up to make -j 4 it seems everything is going well , once I came to ./run_tests I could not have successful test passing. 
The error:

```
dgkernels/ad_dg_convection.exo: Working Directory: /home/projects/moose/test/tests/dgkernels/ad_dg_convection
dgkernels/ad_dg_convection.exo: Running command: /home/projects/moose/test/moose_test-opt -i ad_dg_convection.i Outputs/exodus=false -snes_test_jacobian -snes_force_iteration --error --error-unused --error-override --no-gdb-backtrace
dgkernels/ad_dg_convection.exo: /home/projects/moose/test/moose_test-opt: error while loading shared libraries: libquadmath.so.0: cannot open shared object file: No such file or directory
dgkernels/ad_dg_convection.exo: 
dgkernels/ad_dg_convection.exo: ################################################################################
dgkernels/ad_dg_convection.exo: Tester failed, reason: EXPECTED OUTPUT NOT FOUND
dgkernels/ad_dg_convection.exo: 
dgkernels/ad_dg_convection.exo ............................................ FAILED (EXPECTED OUTPUT NOT FOUND)
```

My question is what is the suitable version of ubuntu for current MOOSE package. 
Although maybe nothing to do with ubuntu version. Maybe problem is elsewhere. 

Thank you in advance.

**Comment by GiudGiud** [comment]
Hello

Ubuntu 20.04 should work well with moose. With the conda install, libquadmath.so should be present in conda/lib.
If you used that way of installing:
if the shared library is present in that directory, I'd say try to run ldconfig on that directory and try again. 

Best,
Guillaume

**Reply by sinaheidari22** [reply]
Thank you for your reply, I think it was the problem with libmesh. Now is solved.

**Reply by GiudGiud** [reply]
Ok thanks for letting us know @sinaheidari22 .
I updated the title to make it easier to search for this particular problem.

**Comment by dschwen** [comment, marked as answer]
```
sudo apt-get install libquadmath0
```

**Reply by sinaheidari22** [reply]
Thank you for your reply.
